### ISSUE TITLE
on Mouse Up, sometimes the Image still move while Mouse move 

### ISSUE THREAD

**Issue by nephon** [hongkhanh/cropbox#19]
Instead of having "mousemove" Event listen for all the time
el.on('mousedown', imgMouseDown);
//el.on('mousemove', imgMouseMove);
$(window).on('mouseup', imgMouseUp);

/*******************************************************************************************/
Adjustment as below
imgMouseDown = function(e)
{
    el.on('mousemove', imgMouseMove);
}

imgMouseUp = function(e)
{
    el.off('mousemove');
}

/***********************************************************************************************/
I have also change from "bind" to "on"

**Comment by sidaurukfreddy** [comment]
Yes me too here, i have mulitple crop in one page the first one working but the second one images still move while mouse move, where is the author

**Comment by sidaurukfreddy** [comment]
hey i can solved this issue just go to cropbox.js

<code>
 imgMouseDown = function(e)
            {
				  e.preventDefault();//add this image will not move wbhile mouse move
                        e.stopImmediatePropagation();
				obj.state.dragable = true;
                obj.state.mouseX = e.clientX;
                obj.state.mouseY = e.clientY;
            },
</code>

**Comment by sidaurukfreddy** [comment]
did you solved this issue ? Thanks

**Comment by 2van** [comment]
@sidaurukfreddy @nephon  I also have this issue, and it has been resolved. see [imgMouseUp sometimes doesn't work](https://github.com/hongkhanh/cropbox/pull/30). the reason may be related with this question [event.stopPropagation() and event.preventDefault().](http://stackoverflow.com/questions/5302903/jquery-event-stopimmediatepropagation-vs-return-false)
